pyproject.toml

```diff
@@ -20,7 +20,7 @@ classifiers = [
 	"Operating System :: OS Independent",
 ]
 requires-python = ">=3.7"
-dependencies = ["slack_sdk>=3.35.0,<4"]
+dependencies = ["slack_sdk==3.37.0"]
```

Comment:
suggestion: We'll also need to change this back 🙂

Reply:
@mwbrooks Done: https://github.com/slackapi/bolt-python/pull/1387/commits/9e5bb32844e35dd01e9763bcf03f869851e674b4.